frontend/src/app/_pipes/chunk-array.pipe.ts

```diff
@@ -0,0 +1,26 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'ChunkArray'
+})
+export class ChunkArrayPipe implements PipeTransform {
```

Comment:
This is not an intuitively named pipe. Never from the tests nor the function, can I quite tell what this does, or how its helpful, especially as it looks like its only actually invoked in one place in the code. If this is just a filter, why not use native es6 / array.map() / array.filter()?

Reply:
@lauraGgit It splits the array into multiple columns, column number passed to the pipe. We are going change the so it's more clear.